src/components/pages/Animations/FaceoffReveal.js

```diff
@@ -191,10 +197,14 @@ const FaceoffReveal = props => {
     history.push('/child/match-up');
   };
 
+  const replayMatchup = event => {
+    window.location.reload();
+  };
+
   return (
-    <div
-      className="FaceoffReveal"
-      style={{ backgroundColor: dynamicBackgroundColor }}
+    <section
+      id="big-reveal"
+      className={`FaceoffReveal bg-${dynamicBackgroundColor}`}
```

Comment:
Adding `full-page` className here should fix the spacing above and below the background color.